src/components/ViewNote/ViewNote.css

```diff
@@ -17,4 +17,61 @@
 
 .card_icons:hover {
     cursor: pointer;
+}
+
+.dropdown {
+    position: relative;
+    display: inline-block;
+}
+
+.dropdown-content{
+  position: absolute;
+  display: flex;
+  flex-flow: row wrap;
+  gap: 10px;
+  background-color: #f9f9f9;
+  min-width: 13rem;
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+  padding: 12px 16px;
```

Comment:
for more than 10px you should use rem

Reply:
Here also i want the exact size irrespective of responsiveness.